src/runtime/runtime.js

```diff
@@ -562,7 +562,10 @@ export class ConfiguredRuntime {
     // Report redirect errors if any.
     this.activityPorts_.onRedirectError(error => {
       this.analyticsService_.addLabels(['redirect']);
-      this.analyticsService_.logEvent(AnalyticsEvent.EVENT_PAYMENT_FAILED);
+      this.eventManager().logSwgEvent(
```

Comment:
I would use this.eventManager_ instead of this.eventManager() since we are in the constructor.